### PR TITLE
[PRTL-2536] show survival plot curve labels in print/svg

### PR DESCRIPTION
**Pull request by samrichca**
lots of eslint changes here, see comments for what I changed

### CODE REVIEW

File: src/packages/@ncigdc/utils/wrapSvg.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ function buildForeignObject({
 }): { html: string, height: number } {
   const foreignObjects = elements.filter(Boolean);
   // $FlowIgnore
-  const elementsHeight = sum(foreignObjects.map(e => e.offsetHeight));
+  const elementsHeight = sum(foreignObjects.map(e => e.offsetHeight || 15));
```
Comment (samrichca) [inline src/packages/@ncigdc/utils/wrapSvg.js]: force the label elements to have a height

File: src/packages/@ncigdc/utils/survivalplot.js [hunk 2]
```diff
@@ -275,20 +312,30 @@ export const getSurvivalCurvesArray = memoize(
...
               }
               : {
                 key: valueName,
                 value: (
                   <span>
-                    S<sub>{i + 1}</sub> (N = {getCaseCount(i)})
+                    S
+                    <sub>{i + 1}</sub>
+                    {` (N = ${getCaseCount(i)})`}
+                    <span className="print-only inline">
```
Comment (samrichca) [inline src/packages/@ncigdc/utils/survivalplot.js]: add labels to clinical analysis survival plot

File: src/packages/@ncigdc/components/SurvivalPlotWrapper.js [hunk 3]
```diff
@@ -95,103 +96,110 @@ const SurvivalPlotWrapper = ({
...
+                  className: CLASS_NAME,
+                  embed: {
+                    top: {
+                      elements: legend
+                        .map((l, i) => {
+                          const legendItem = document.querySelector(
+                            `.${uniqueClass} .legend-${i}`
+                          ).cloneNode(true);
+                          const legendTitle = legendItem.querySelector('span.print-only.inline');
+                          if (legendTitle !== null) legendTitle.className = '';
```
Comment (samrichca) [inline src/packages/@ncigdc/components/SurvivalPlotWrapper.js]: make the print-only label visible in svg/png

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard.js [hunk 4]
```diff
@@ -766,37 +817,35 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
...
-      className="clinical-analysis-categorical-card"
-    >
+      >
       <Row
         style={{
           justifyContent: 'space-between',
           alignItems: 'center',
           margin: '5px 0 10px',
         }}
-        id={wrapperId}
```
Comment (samrichca) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard.js]: removing this duplicate ID fixes the histogram download issue

**Comment by caravinci** [comment]
Yeah! they're super useful!

**Pull request merged**